unitex/src/fr/umlv/unitex/frames/ConcordanceParameterFrame.java

```diff
@@ -599,6 +599,7 @@ void buildConcordance(boolean onlyAmbiguous) {
 		}
 		GlobalProjectManager.search(null)
 				.getFrameManagerAs(InternalFrameManager.class).closeConcordanceFrame();
+		System.out.println("locate :"+command.getCommandLine());
```

Comment:
Check if it's something you really want to do...

Reply:
i forgot to remove it